gnucash/gnome-utils/dialog-utils.c

```diff
@@ -535,6 +535,11 @@ gnc_handle_date_accelerator (GdkEventKey *event,
             g_date_to_struct_tm (&gdate, tm);
         return TRUE;
 
+    case GDK_KEY_question:
+        /* Translators: the []-+MHYRT keyboard shortcuts are hard coded
+           and are not localizable */
+        gnc_info_dialog (NULL, "%s", _("This is a date field. Keyboard shortcuts are available: \n- previous day\n+ next day\nshift- previous week\nshift+ next week\n[,alt- previous month \n],alt+ next month\nctrl- previous year\nctrl+next year\nM beginning of month\nH end of month\nY beginning of year\nR end of year\nT today"));
+        return TRUE;
```

Comment:
Here's tooltip- is this better?

![image](https://user-images.githubusercontent.com/1975870/93691846-b386ec80-fada-11ea-92de-48aa308db2a7.png)
![image](https://user-images.githubusercontent.com/1975870/93691852-c0a3db80-fada-11ea-9bbf-405a576789f1.png)

Reply:
```
        gnc_info_dialog (NULL, "%s", _("This is a date field. Keyboard shortcuts are available:\n\
                                        [-]: previous day\n\
                                        [+]: next day\n\
                                        [shift]+[-]: previous week\n\
                                        [shift]+[+]: next week\n\
                                        [alt]+[-] or "[": previous month \n\
                                        [alt]+[+] or "]": next month\n\
                                        [ctrl]+[-]: previous year\n\
                                        [ctrl]+[+]: next year\n\
                                        [M]: beginning of month\n\
                                        [H]: end of month\n\
                                        [Y]: beginning of year\n\
                                        [R]: end of year\n\
                                        [T]: today"));

```
would be easier to read for
1. coders by the line breaks
2. users by marking keys
Note: on a german keyboard "[" is [AltGr]+[8] and "]" [AltGr]+[9]

Ideally the Characters would be translatable e.g. for de to
M, T, (Monat)
J, R (Jahr)
H (heute)

Reply:
the MHYRT hotkeys are a direct descendant of Quicken -- you can test via https://archive.org/details/msdos_Quicken_7 and also I'd argue very difficult to translate.

Reply:
… in the english version.

They are the first and the last character of the corresponding word like "M" and "H" in "MontH".
And it is in the responsibility of the translator to choose another in case of duplicates.

BTW, Fix:
```
                                        [alt]+[-] or \"[\": previous month\n\
                                        [alt]+[+] or \"]\": next month\n\

```